code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -197,6 +197,7 @@
 	var/regeneration_multiplier = 1
 	var/speed_modifier = 0
 	var/phero_modifier = 0
+	var/received_phero_caps = list()
```

Comment:
instead of a list, i reckon you can just make a var for each phero type and put the cap there. then, when a xeno gets its pheremone added, you can do min(phere_cap, phere_received)

Reply:
yeah i doubt a lot of xenos would have limited pheros ima keep it as is